### ISSUE TITLE
Error when executing: "function Y_Env_Init[2]..Y_ServerStart"

### ISSUE THREAD

**Issue by rocketmagnet83** [JBakamovic/yavide#76]
i've recompiled vim with python3 and python2 support and did all steps from the wiki to install (just run ./install.sh), ihave libclang installed and clang but when i run yavide i get a bunch of errors:

Fehler beim Ausführen von "function Y_Env_Init[2]..Y_ServerStart":
Zeile    1:
E319: Der Befehl ist in dieser Version nicht implementiert: python << EOF
Betätigen Sie die EINGABETASTE oder geben Sie einen Befehl ein
Fehler beim Ausführen von "function Y_Env_Init[2]..Y_ServerStart":
Zeile    2:
E492: Kein Editorbefehl: from server.yavide_server import yavide_server_run
Betätigen Sie die EINGABETASTE oder geben Sie einen Befehl ein
Fehler beim Ausführen von "function Y_Env_Init[2]..Y_ServerStart":
Zeile    3:
E492: Kein Editorbefehl: from multiprocessing import Process
Betätigen Sie die EINGABETASTE oder geben Sie einen Befehl ein
Fehler beim Ausführen von "function Y_Env_Init[2]..Y_ServerStart":
Zeile    5:
E492: Kein Editorbefehl: server = Process(target=yavide_server_run, args=(server_queue, vim.eval('v:servername')), name="yavide_server")

**Comment by JBakamovic** [comment]
What is the output of `gvim --version | grep python`? Make sure you run this on the version you've built for yourself.

**Comment by ChaoticConundrum** [comment]
I think I have a similar error, I see this each time on startup of yavide:
![spectacle t17644](https://user-images.githubusercontent.com/2348393/31414443-ee3c8eb4-adeb-11e7-8d1d-cf99e74874f7.png)

`gvim --version`:

    VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Oct  7 2017 13:50:55)
    Included patches: 1-1176
    Compiled by Arch Linux
    Huge version with GTK3 GUI.  Features included (+) or not (-):
    +acl             +file_in_path    +mouse_sgr       +tag_old_static
    +arabic          +find_in_path    -mouse_sysmouse  -tag_any_white
    +autocmd         +float           +mouse_urxvt     +tcl/dyn
    +balloon_eval    +folding         +mouse_xterm     +termguicolors
    +browse          -footer          +multi_byte      +terminal
    ++builtin_terms  +fork()          +multi_lang      +terminfo
    +byte_offset     +gettext         -mzscheme        +termresponse
    +channel         -hangul_input    +netbeans_intg   +textobjects
    +cindent         +iconv           +num64           +timers
    +clientserver    +insert_expand   +packages        +title
    +clipboard       +job             +path_extra      +toolbar
    +cmdline_compl   +jumplist        +perl/dyn        +user_commands
    +cmdline_hist    +keymap          +persistent_undo +vertsplit
    +cmdline_info    +lambda          +postscript      +virtualedit
    +comments        +langmap         +printer         +visual
    +conceal         +libcall         +profile         +visualextra
    +cryptv          +linebreak       +python/dyn      +viminfo
    +cscope          +lispindent      +python3/dyn     +vreplace
    +cursorbind      +listcmds        +quickfix        +wildignore
    +cursorshape     +localmap        +reltime         +wildmenu
    +dialog_con_gui  +lua/dyn         +rightleft       +windows
    +diff            +menu            +ruby/dyn        +writebackup
    +digraphs        +mksession       +scrollbind      +X11
    +dnd             +modify_fname    +signs           -xfontset
    -ebcdic          +mouse           +smartindent     +xim
    +emacs_tags      +mouseshape      +startuptime     -xpm
    +eval            +mouse_dec       +statusline      +xsmp_interact
    +ex_extra        +mouse_gpm       -sun_workshop    +xterm_clipboard
    +extra_search    -mouse_jsbterm   +syntax          -xterm_save
    +farsi           +mouse_netterm   +tag_binary

Python 2 and 3 installed from Arch Linux repositories.

**Comment by JBakamovic** [comment]
Like it says in the error message, `watchdog` Python module is missing. Please try to install it via pip or system package manager and try to run `yavide` again. Though it should have been installed by the `install.sh` script. Many users for some reason miss this exact package.

**Comment by ChaoticConundrum** [comment]
Yes, that did it. Thank you.

**Comment by JBakamovic** [comment]
@ChaoticConundrum Glad you fixed it. Have you got any idea why this package wouldn't have been installed by the `install.sh` script? Did you notice any error during the installation process? 

I've got plenty of such issues reported lately by different people. For me on Fedora it's working flawlessly but seems that is not the case on other distros. Which is strange because that package is being installed through `pip` which should be more unified way to install python dependencies ...

**Comment by ChaoticConundrum** [comment]
Actually, yes. It looks like `install.sh` installs `python` and `python-pip`, which on Arch Linux, are Python 3. From the packages installed for other distros, I am guessing you are trying to install Python 2? On Arch Linux, the Python 2 packages are `python2` and `python2-pip`, and the `pip` command is `pip2`.

**Comment by JBakamovic** [comment]
Yes, that's correct. I am trying to install Python2 dependencies. I suppose I should try with `pip2` instead. Thanks for your input.

**Comment by chaox1013** [comment]
I seem to only be getting this issue when I open a file using yavide., i.e. "yavide sample.cpp".
When I open yavide on its own, then open any file I never run into this issue. I have pip, pip2, pip2.7, pip3, pip3.5 and I already updated all of them and it doesn't seem to help.
